app/src/main/java/com/monstarlab/core/extensions/Extensions.kt

```diff
@@ -0,0 +1,9 @@
+package com.monstarlab.core.extensions
+
+inline fun <reified T> Any.castAs(): T {
```

Comment:
Do we really need extensions to cast? I think most people will forget this exists

Reply:
I find it useful 
Its much easier than doing `(stuf as Something).doStuff` 
doesnt hurt it being here i think